packages/patoline/patoline.0.2/opam

```diff
@@ -0,0 +1,56 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Patoline typesetting system and libraries"
+description: """
+Patoline is a collection of typesetting libraries designed to be combined
+   into a variety of possible applications. Its primary goal is not to offer
+   the definitive answer to all typesetting problems, but rather to write the
+   painful and boring parts of the job, so that interesting tools could be
+   written easily. Obviously, a related (but much smaller) project is the
+   Patoline compiler, which compiles a mixed Wiki/LaTeX/Ocaml syntax into a
+   variety of output formats, including traditional PDF files, but also web
+   servers that deliver dynamic contents."""
+maintainer: ["Rodolphe Lepigre <lepigre@mpi-sws.org>"]
+authors: [
+  "Pierre-Étienne Meunier"
+  "Christophe Raffalli <christophe@raffalli.eu>"
+  "Rodolphe Lepigre <lepigre@mpi-sws.org>"
+  "Tom Hirschowitz"
+  "Florian Hatat <patoline@hatat.me>"
+]
+license: "GPL-2.0"
+homepage: "https://patoline.github.io"
+bug-reports: "https://github.com/patoline/patoline/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.7" & build}
```

Comment:
Dune should not have the build tag due to forward-compatibility reasons. Since this is a binary it should not be a problem but it is better to stay on the safe side. If/once this can be re-introduced, we will do it manually on the opam repository for all the affected packages

Reply:
OK, I had no idea! I fixed this in https://github.com/patoline/patoline/commit/75fd8c928efc68d0aaa400d3a699a0e668c26c5f but I'll not re-tag since we already have a fix here.